pyteal/config.py

```diff
@@ -1,8 +1,14 @@
+from algosdk.atomic_transaction_composer import ABI_RETURN_HASH
+
+
 # Maximum size of an atomic transaction group.
 MAX_GROUP_SIZE = 16
 
 # Number of scratch space slots available.
 NUM_SLOTS = 256
 
 # Method return selector in base16
-RETURN_METHOD_SELECTOR = "0x151F7C75"
+RETURN_HASH_PREFIX = ABI_RETURN_HASH
+
+# Method argument number limit
+METHOD_ARG_NUM_LIMIT = 15
```

Comment:
nit: rename to `METHOD_ARG_TUPLE_CUTOFF` or similar? "Limit" implies this is the maximum

Reply:
cutoff sounds good